tests/avocado/test_ovmf_with_240_vcpus.py

```diff
@@ -0,0 +1,69 @@
+import os
+import shutil
+import sys
+
+from avocado import main
+from avocado_qemu import test
+
+
+class TestOvmfVcpus(test.QemuTest):
+    """
+    Run with:
+
+        avocado run test_ovmf_with_240_vcpus.py \
+        -m test_ovmf_with_240_vcpus.py.data/parameters.yaml
+
+    :avocado: enable
+    :avocado: tags=ovmf
+    """
+
+    def setUp(self):
+        ovmf_code_path = self.params.get('OVMF_CODE',
+                                         default='/usr/share/edk2/ovmf/OVMF_CODE.secboot.fd')
+        ovmf_vars_path = self.params.get('OVMF_VARS',
+                                         default='/usr/share/edk2/ovmf/OVMF_VARS.fd')
+        if not ovmf_code_path or not os.path.exists(ovmf_code_path):
+            basename = os.path.basename(__file__)
+            self.cancel('OVMF_CODE file not found. Set the correct '
+                        'path on "%s.data/parameters.yaml" and run this test '
+                        'with: "avocado run %s -m %s.data/parameters.yaml"' %
+                        (basename, basename, basename))
+        if not ovmf_vars_path or not os.path.exists(ovmf_vars_path):
+            basename = os.path.basename(__file__)
+            self.cancel('OVMF_VARS file not found. Set the correct '
+                        'path on "%s.data/parameters.yaml" and run this test '
+                        'with: "avocado run %s -m %s.data/parameters.yaml"' %
+                        (basename, basename, basename))
+
+        ovmf_code_tmp = os.path.join(self.teststmpdir,
+                                     os.path.basename(ovmf_code_path))
+        ovmf_vars_tmp = os.path.join(self.teststmpdir,
+                                     os.path.basename(ovmf_vars_path))
+        if not os.path.exists(ovmf_code_tmp):
+            shutil.copy(ovmf_code_path, self.teststmpdir)
```

Comment:
aren't you afraid other tests might modify the file? I'd use `tmpdir` at least for `vars` file. Although as the `code` file should be usually added `ro`, I'd only copy the `vars` file and use the `code` file directly (and add the `ro` argument)